server/integration_tests/test_data/e2e_date_range/whichcountriesinafricahavehadthegreatestincreaseinelectricityaccessoverthelast10years/chart_config.json

```diff
@@ -19,7 +19,7 @@
                       "country/DZA"
                     ],
                     "lineTileSpec": {
-                      "endDate": "2023",
+                      "endDate": "2024",
```

Comment:
Is this new data?

Reply:
this was from the inclusive/exclusive date range change earlier this week (last 10 years should include the current year)